### DISCUSSION TITLE
Help with code please

### DISCUSSION THREAD

**Discussion by pakhomovvv** [Cyfrin/foundry-full-course-cu#2490] [Q&A]
```solidity
// SPDX-License-Identifier: MIT

pragma solidity 0.8.18;

import {Script} from "forge-std/Script.sol";
import {SimpleStorage} from "../src/SimpleStorage.sol";

contract DeploySimpleStorage is Script {
    function run() external returns (SimpleStorage) {
        vm.startBroadcast();

        SimpleStorage simpleStorage = new SimpleStorage();

        vm.stopBroadcast();
        return simpleStorage;
    }
}
```

this is my code, in the 5 line there is an error, which is:
"Source "forge-std/Script.sol" not found: File import callback not supported(6275)"
Whats happening?

**Comment by guschi18** [comment]
Go to your lib folder and check if forge-std is installed. If not, you have to install it and the error should be away.

**Reply by pakhomovvv** [reply]
Hi. thanks, but something weird happened; yesterday the DeploySimpleStorage.s.sol was in red (the error that i was talking ab), but today when i entered again to VSC, the issue has been solved (idk how). Btw, thank you

**Reply by EngrPips** [reply]
Interesting, Sometimes `vscode` loads slowly and unnecessarily marks stuff as red

**Comment by EngrPips** [comment]
Hello @pakhomovvv, Please do as @guschi18 advise and let's see if it help you resolve the issue.

**Comment by Sachimugu** [comment]
In case anyone else runs into the above error, confirm that you have the ```forge-std``` folder in your ```lib``` directory. Then, restart your VS Code, and the error should disappear

**Reply by EngrPips** [reply]
Great heads-up.
